src/frontend/src/pages/ChangeRequestDetailPage/WorkPackageComparisonBlock.tsx

```diff
@@ -0,0 +1,117 @@
+import { ChangeRequest, StandardChangeRequest } from 'shared';
```

Comment:
Could you do some abstraction here for the common comparisons between this and `ProjectComparisonBlock`

Reply:
not sure if i can, for calling the wbs' initial values its fine cus those come from the change request. but for the proposed changes, its either calling it from the hook for project proposed changes or the other hook for work package proposed changes which is why I split them into two separate files. all the common comparisons contain calls to these hooks, and since I cant call both hooks together I think I have to hardcode the common things in each file. if there's another way lmk and ill get on it.

Reply:
I'm assuming you meant the reverse for initial vs proposed (unless I'm misunderstanding) - the common fields are shared using the wbs element interface which can be passed in instead of specifying a project or work package.

So you would have a wbs comparison block that is used by both project and work package